bot/code_review_bot/report/phabricator.py

```diff
@@ -191,7 +191,9 @@ def comment_inline(self, revision, issue, existing_comments=[]):
         comment = {
             "diffID": revision.diff_id,
             "filePath": issue.path,
-            "lineNumber": issue.line,
+            "lineNumber": issue.line
+            if issue.line is not None
+            else 1,  # support full file
```

Comment:
Did you try this out on a sample Phabricator diff? Could you share the link? (I'm curious to see what it looks like, does it expand the whole file to show the comment?)

Reply:
I tried with 0 [here](https://phabricator.services.mozilla.com/D52765#1611466) but Phabricator did not display the inline comment (still acepting it). So i'm using 1, so it should show up on the first line

Reply:
Here is a sample output with current code: https://phabricator-dev.allizom.org/D1535#inline-20255